.travis.yml

```diff
@@ -49,11 +49,11 @@ matrix:
 
     - os: linux
       env:
-        - TESTSUITE=gcs_service
+        - TESTSUITE=gcs_service_disabled
```

Comment:
Perhaps we no longer need these tests in travis?

Reply:
These tests in travis is used to avoid code modification affecting GCS service disable scenario. Nowwe can still use the environment variable to disable gcs service.